db/schema.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# This file is the source Rails uses to define your schema when running `bin/rails
+# db:schema:load`. When creating a new database, `bin/rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema[7.0].define(version: 20_230_312_100_332) do
+  # These are extensions that must be enabled in order to support this database
+  enable_extension 'plpgsql'
+
+  create_table 'users', force: :cascade do |t|
+    t.string 'first_name', default: '', null: false
+    t.string 'last_name', default: '', null: false
+    t.string 'email', default: '', null: false
+    t.string 'user_name', default: '', null: false
+    t.string 'encrypted_password', default: '', null: false
+    t.string 'phone', default: '', null: false
+    t.integer 'role', default: 0, null: false
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+    t.index ['email'], name: 'index_users_on_email', unique: true
+    t.index ['user_name'], name: 'index_users_on_user_name', unique: true
+  end
+end
```

Comment:
Can you please try to fill up the seed file to create some dummy users with factorybot so that when running the project some default users will be present in the database through the seed file? Thank you!

Reply:
seed added for user model.